app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -79,6 +84,7 @@ public void validate(Object obj, Errors errors) {
         validateEmails(errors, study.getSupportEmail(), "supportEmail");
         validateEmails(errors, study.getTechnicalEmail(), "technicalEmail");
         validateEmails(errors, study.getConsentNotificationEmail(), "consentNotificationEmail");
+        validateDataGroupNamesAndFitForSynapseExport(errors, study.getDataGroups());
```

Comment:
Do we also want to validate taskIdentifiers? Or are those entirely unrestricted?